### PR TITLE
utils_test/libvirt: Add a function to set disk attributes

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/libvirt_xml/vm_xml.py [hunk 1]
```diff
@@ -802,6 +802,26 @@ def get_disk_attr(vm_name, target, tag, attr, virsh_instance=base.virsh):
...
+
+        :param vm_name: domain name
+        :param target: dev of the disk
+        :param tag: disk tag
+        :param attr: the tag attribute dict to set
+        """
+        key = ""
+        try:
+            disk = self.get_disk_all()[target]
+            if tag in ["driver", "boot", "address", "alias", "source"]:
```
Comment (will-Do) [inline virttest/libvirt_xml/vm_xml.py]: There are many disk attributes , you can check ```__slots__``` in libvirt_xml/devices/disk.py
And I prefer to add these kind of functions to utils_test/libvirt.py as the xml related modules here already a bit of mess.

**Comment by dzhengfy** [comment]
@will-Do , move it to libvirt.py. Please check again.

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -1624,6 +1624,28 @@ def file_exists():
     return diskxml.xml
 
 
+def set_disk_attr(vmxml, target, tag, attr):
+    """
+    Set value of disk tag attributes for a given target dev.
+    :param vmxml: domain VMXML instance
+    :param target: dev of the disk
+    :param tag: disk tag
+    :param attr: the tag attribute dict to set
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: Better to add 'return':
```
:return xx: xxx
```

Comment (dzhengfy) [inline virttest/utils_test/libvirt.py]: @will-Do , updated and please check again.Thanks.

**Pull request merged**